fix(Popover): Fix scrollposition for FF

chvonrohr opened the pull request:
fixes #357

srvance commented on addon/utils/get-position.js:
```diff
@@ -17,8 +17,8 @@ export default function getPosition(el) {
   let elOffset = isBody ? { top: 0, left: 0 } : (isSvg ? {} : {
-    top: rect.top + document.body.scrollTop,
-    left: rect.left + document.body.scrollLeft
+    top: rect.top + (window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0),
```
Might be worth extracting these into functions and documenting the browser dependencies that contribute to it.

chvonrohr replied:
might be a good idea... two lines below scroll-position is also evaluated. But I don't know where it's used...

srvance commented on addon/utils/get-position.js:
```diff
@@ -35,3 +36,12 @@ export default function getPosition(el) {
+function getScrollPosition() {
+  return {
+    //    IE, Firefox                            Chrome
+    top:  document.documentElement.scrollTop  || document.body.scrollTop  || 0,
```
What happened to `window.page[XY]Offset`?

chvonrohr replied:
I was checking which browsers are targeted by ```window.page[XY]Offset``` but couldn't find a clear answer. As most seem to work with the above two, I added them for now. Tested in FF, Safari, Chrome

srvance commented:
Looks like ESLint issues, so should be simple to fix.

simonihmig commented:
@chvonrohr Thanks for your contribution! Happy to merge this once the ESLint issues and https://github.com/kaliber5/ember-bootstrap/pull/359#discussion_r121978671 have been resolved!

simonihmig commented:
@chvonrohr Thanks for fixing the linting errors! What about `window.pageYOffset`, is this not needed anymore?

chvonrohr commented:
@simonihmig Seems that ```document.documentElement.scrollTop``` and ```document.body.scrollTop``` cover all major browsers. I couldn't find why its needed, so I removed this one. I added it in the first commit, as it was mentioned in https://stackoverflow.com/questions/28633221/document-body-scrolltop-firefox-returns-0-only-js (see #357).

simonihmig commented:
Ok, LGTM. @srvance your review seems to have been addressed. Ready to go?
